tests/test_interface.py

```diff
@@ -100,13 +110,12 @@ def tearDown(self):
 
 
   def test_rsa(self):
-    """Test RSA key generation and import interface functions. """
+    """Test RSA key _generation and import interface functions. """
 
     # TEST: Generate default keys and import
     # Assert location and format
     fn_default = "default"
-    fn_default_ret = generate_and_write_rsa_keypair(
-        filepath=fn_default, password="")
+    fn_default_ret = _generate_and_write_rsa_keypair(filepath=fn_default)
```

Comment:
Unencrypted, correct?

Reply:
Correct.

Reply:
But note that this is the protected internal `_generate_and_write_rsa_keypair`  (see `_`-prefix)  function for flexible but hazardous use. The public `generate_and_write_rsa_keypair` does not accept an empty password, and thus can not write unencrypted keys.